st2reactor/st2reactor/bootstrap/ruletypesregistrar.py

```diff
@@ -0,0 +1,80 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+
+from st2common import log as logging
+from st2common.constants.rules import RULE_TYPE_STANDARD, RULE_TYPE_BACKSTOP
+from st2common.models.api.rule import RuleTypeAPI
+from st2common.persistence.rule import RuleType
+
+__all__ = [
+    'register_rule_types',
+    'RULE_TYPES'
+]
+
+
+LOG = logging.getLogger(__name__)
+
+RULE_TYPES = [
+    {
+        'name': RULE_TYPE_STANDARD,
+        'description': 'standard rule that is always applicable.',
+        'enabled': True,
+        'parameters': {
+        }
+    },
+    {
+        'name': RULE_TYPE_BACKSTOP,
```

Comment:
It is a baseball analogy so used backstop which is sort the right real work example. I am cool with catchall but find it less expressive than backstop and which is why I used this name. But you know democracy and all so I would put it to a vote and see what others prefer.

```
back·stop
ˈbakstäp/
noun
1.
a person or thing placed at the rear of or behind something as a barrier, support, or reinforcement.
"bullets volleyed into the backstop of a flood-control canal"
verb
1.
BASEBALL
act as backstop for.
```